### ISSUE TITLE
Why is the saliency-score negative?

### ISSUE THREAD

**Issue by jianfeng0369** [line/lighthouse#47]
There is no problem downloading the code, configuring the environment, running the code, uploading the video, and outputting the results, but why is the saliency-score negative? Is this normal?

Thanks for your answer! Your work is very meaningful!

![2024-11-27](https://github.com/user-attachments/assets/6cf23cbe-4cfc-42a7-91bb-cd1062c9ab47)

**Comment by awkrail** [comment]
@jianfeng0369 
This is expected.
Hi, as mentioned in moment detr paper, the loss function of saliency score is as follow:
<img width="1024" alt="スクリーンショット 2024-11-28 15 55 10" src="https://github.com/user-attachments/assets/f2597434-873e-4e4d-bbbc-343d78b83d95">
S(.) is a neural network and there is no constraints of the scores in terms of positiveness/negativeness. The model is trained for positive clips (salient clips) having better scores than negative clips (less-salient clips). The trained models have negative values, but the sign of values itself does not have any meaning.

**Comment by jianfeng0369** [comment]
> @jianfeng0369 This is expected. Hi, as mentioned in moment detr paper, the loss function of saliency score is as follow: <img alt="スクリーンショット 2024-11-28 15 55 10" width="1024" src="https://private-user-images.githubusercontent.com/19744782/390673136-f2597434-873e-4e4d-bbbc-343d78b83d95.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI3OTA5NDgsIm5iZiI6MTczMjc5MDY0OCwicGF0aCI6Ii8xOTc0NDc4Mi8zOTA2NzMxMzYtZjI1OTc0MzQtODczZS00ZTRkLWJiYmMtMzQzZDc4YjgzZDk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI4VDEwNDQwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYxZTQ4Mzc0NDA2MDMyYzhmOTU2Njg4YWViMWMwMDViNWRjZWU4YjVjYjBhNWVjOTVjMzU4MWU3MmY1MmRkZjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.hQOL-0k2OsvK0xky-G_nseq-IDzZQdXfuaQrKTUM3dM"> S(.) is a neural network and there is no constraints of the scores in terms of positiveness/negativeness. The model is trained for positive clips (salient clips) having better scores than negative clips (less-salient clips). The trained models have negative values, but the sign of values itself does not have any meaning.

Thank you very much for your reply! I learned a lot from your answers and papers.
